packages/apollo-language-server/src/languageProvider.ts

```diff
@@ -178,9 +178,7 @@ export class GraphQLLanguageProvider {
         if (!suggestedField) {
           return suggest;
         } else {
-          const requiredArgs = suggestedField.args.filter(
-            a => a.type instanceof GraphQLNonNull
-          );
+          const requiredArgs = suggestedField.args.filter(isNonNullType);
```

Comment:
Is this right, or would you need to check the args' `arg.type` like the old check was doing?

Reply:
Good catch!